yarn.lock

```diff
@@ -337,7 +337,7 @@
 
 "@electron/get@^2.0.0":
   version "2.0.2"
-  resolved "https://registry.npmjs.org/@electron/get/-/get-2.0.2.tgz#ae2a967b22075e9c25aaf00d5941cd79c21efd7e"
+  resolved "https://registry.yarnpkg.com/@electron/get/-/get-2.0.2.tgz#ae2a967b22075e9c25aaf00d5941cd79c21efd7e"
```

Comment:
@sprocketc Looks like these were changed in https://github.com/logseq/logseq/pull/9482 so reverting them back

Reply:
@logseq-cldwalker I think `registry.yarnpkg.com` is now just a [cname](https://toolbox.googleapps.com/apps/dig/#CNAME/registry.yarnpkg.com) that points to `npmjs.org`, so those should be the same thing (see https://yarnpkg.com/getting-started/qa#why-registryyarnpkgcom-does-facebook-track-us)
If we want to always use `registry.yarnpkg.com` for consistency, I'll have to make some additional changes, but I don't think it's worth it.